src/modules/mc_att_control/mc_att_control_main.cpp

```diff
@@ -166,6 +168,7 @@ class MulticopterAttitudeControl
 	struct battery_status_s				_battery_status;	/**< battery status */
 	struct sensor_gyro_s			_sensor_gyro;		/**< gyro data before thermal correctons and ekf bias estimates are applied */
 	struct sensor_correction_s		_sensor_correction;		/**< sensor thermal corrections */
+	struct sensor_corrected_s		_sensor_corrected;		/**< sensor in-run bias corrections */
```

Comment:
@dagar sensor_correction and sensor_corrected is a terrible choice - could we change to sensor_bias instead? Because that is what it really is.

Reply:
That came in after what I originally did, but it sounds fine to me. There are a few users of the bias correct accel though. 
I understand why we're doing it, but this entire sensor reconstruction in the rate controller feels awkward. I'll do the bias rename for now, but I'd like to discuss other options later on.